src/lesson10/exceprions/userValidationService/UserValidationException.java

```diff
@@ -0,0 +1,8 @@
+package lesson10.exceprions.userValidationService;
+
+public class UserValidationException{
+
+    public void exception() {
```

Comment:
I haven't seen such way of using exceptions. You should extend from either `Exception` or `RuntimeException`
No sense in this method

Reply:
ok. fixed this code